dotCMS/src/main/java/com/dotcms/rest/BundleResource.java

```diff
@@ -160,7 +161,8 @@ public Response getPublishQueueElements(@PathParam("bundleId") final String bund
             if (UtilMethods.isSet(queueElements)) {
                 detailedAssets = publishQueueElementTransformer.transform(queueElements);
             } else {
-                final PublishAuditStatus publishAuditStatus = PublishAuditAPI.getInstance()
+                final int limit = UtilMethods.isSet(limitParam) ? limitParam : NO_LIMIT_ASSETS;
```

Comment:
we have this everywhere, we should create something such as UtilMethods.isSetOrGet(limitParam, NO_LIMIT_ASSETS)

Reply:
done https://github.com/dotCMS/core/pull/21384/commits/855c672916721e9296f4a33af272cebc419b2d62